common/src/Microsoft.Azure.IIoT.Hub.Processor/src/Services/EventProcessorFactory.cs

```diff
@@ -17,6 +17,7 @@ namespace Microsoft.Azure.IIoT.Hub.Processor.Services {
     using System.Diagnostics;
     using Autofac;
     using Microsoft.Azure.IIoT.Exceptions;
```

Comment:
Please move the Exceptions using over using Serilog while you are touching the file.